modules/storages/app/views/storages/project_settings/_toast_no_storage_set_up.html.erb

```diff
@@ -0,0 +1,12 @@
+<!-- Standard Ruby view, please see the controller for comments -->
+<div class="op-toast -info">
+  <div class="op-toast--content">
+    <p>
+      <strong><%= t(:'storages.instructions.no_storage_set_up') %></strong>
+    </p>
+    <p>
+      <%= t(:"storages.instructions.setting_up_additional_storages") %>
+      <%= link_to "#{t(:label_administration)} / #{t(:project_module_storages)}", controller: '/storages/admin/storages', action: :index %>.
```

Comment:
A link will be displayed even though the user might not be permitted to follow that link as that user might not be an admin.

Reply:
Done in b6fcb1aee4.